docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,266 @@
+
+To create the informational tile-only listing's scaffolding:
+
+1. Make sure you're inside the `marketplace` directory:
+   {{< code-block lang="shell" >}}cd $HOME/dd/marketplace{{< /code-block >}}
+2. Run the `ddev` command with the `-t tile` option:
+   {{< code-block lang="shell" >}}ddev create -t tile "<Offering Name>"{{< /code-block >}}
+
+[101]: https://github.com/Datadog/marketplace
+[102]: https://docs.datadoghq.com/developers/integrations/marketplace_offering
+[103]: https://docs.datadoghq.com/developers/integrations/python
+
+{{% /tab %}}
+
+{{% tab "Out-of-the-box Offerings" %}}
+
+## Set up a directory and fork the `integrations-extras` repository
+
+
+1. Create a `dd` directory:
+
+   {{< code-block lang="shell" >}}mkdir $HOME/dd{{< /code-block >}}
+   
+   The Datadog Development Toolkit expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
+
+2. Clone the `integrations-extras` repository:
+
+   {{< code-block lang="shell" >}}git clone git@github.com:DataDog/integrations-extras.git{{< /code-block >}}
+
+## Install and configure the Datadog development toolkit
+
+The Agent Integration Developer Tool allows you to create scaffolding when you are developing an integration by generating a skeleton of your integration tile's assets and metadata. For instructions on installing the tool, see [Install the Datadog Agent Integration Developer Tool][101].
+
+After you install the Developer tool, configure it for the `integrations-extras` repo:
+
+Set `integrations-extras` as the default working repository:
+
+{{< code-block lang="shell" >}}
+ddev config set extras $HOME/dd/integrations-extras
+ddev config set repo extras
+{{< /code-block >}}
+
+If you used a directory other than `$HOME/dd` to clone the integrations-extras directory, use the following command to set your working repository:
+
+{{< code-block lang="shell" >}}
+ddev config set extras <PATH/TO/INTEGRATIONS_EXTRAS>
+ddev config set repo extras
+{{< /code-block >}}
+
+## Populate the integration tile scaffolding
+
+For Datadog API integrations and custom dashboard widgets that will be available out-of-the-box on the [Integrations page][102], use the Datadog Development Toolkit to create scaffolding for an informational tile-only listing.
+
+1. Make sure you're inside the `integrations-extras` directory:
+   {{< code-block lang="shell" >}}cd $HOME/dd/integrations-extras{{< /code-block >}}
+2. Run the `ddev` command with the `-t tile` option
+   {{< code-block lang="shell" >}}ddev create -t tile "<Offering Name>"{{< /code-block >}}
+
+[101]: https://docs.datadoghq.com/developers/integrations/python
+[102]: https://github.com/Datadog/integrations-extras
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Complete the necessary integration asset files
+
+Make sure that the following required assets for your integration are complete:
+
+{{% integration-assets %}}
```

Comment:
Not sure how to leave a comment on this table but this sounds odd
<img width="816" alt="Screen Shot 2023-04-25 at 8 44 44 AM" src="https://user-images.githubusercontent.com/109224841/234331443-4afa7a41-e624-44c8-8fca-54fe4a24c830.png">

Reply:
@alai97 can you help with updating this?

Reply:
Ahh yes! So sorry I missed this comment. @eho1307 would you want to update it to "definition" instead? Sorry, I default to front matter because of Hugo (https://gohugo.io/content-management/front-matter/).

Reply:
Erica's OOO, but yes let's change it to definition! Thank you